src/diffusers/models/attention_processor.py

```diff
@@ -116,6 +116,8 @@ def __init__(
         super().__init__()
         self.inner_dim = out_dim if out_dim is not None else dim_head * heads
         self.query_dim = query_dim
+        self.use_bias = bias
+        self.is_cross_attention = cross_attention_dim is not None
```

Comment:
cc @sayakpaul 
the change here looks good to me - can you take a look and confirm if it's ok?